### PR TITLE
docs: Add Setup Instructions for Windows

### PR DESCRIPTION
**Pull request by KartikSoneji**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation

## Description
Add Windows specific commands to setup instructions.

### CODE REVIEW

**Comment by chronark** [comment]
please merge main, I fixed some stuff in this file earlier

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -48,9 +50,21 @@ Push the database schema to Planetscale:
...
+set DRIZZLE_DATABASE_URL=mysql://{user}:{password}@{host}/{db}
+pnpm drizzle-kit push:mysql
+set DRIZZLE_DATABASE_URL=
+```
+
+#### Mac/Linux
+
 ```sh-session
 cd internal/db
-DRIZZLE_DATABASE_URL='mysql://{user}:{password}@{host}/{db}?ssl={"rejectUnauthorized":true}' pnpm drizzle-kit push:mysql
```
Comment (chronark) [inline .github/CONTRIBUTING.md:53]: why did you remove it?
last time I checked it was required

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -68,8 +82,20 @@ Unkey uses itself to manage its own API keys. To bootstrap the app, run the foll
 
 You need to provide the database credentials as well as the organization ID from clerk as `TENANT_ID`
 
+### Windows
```
Comment (chronark) [inline .github/CONTRIBUTING.md:85]: Let's use tabs here, the default should be Max/linux

https://mintlify.com/docs/content/components/tabs

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -48,9 +50,21 @@ Push the database schema to Planetscale:
 
 > Make sure you replace `user`, `password`, `host` and `db` with your own values
 
+#### Windows
```
Comment (chronark) [inline .github/CONTRIBUTING.md:53]: see below

https://mintlify.com/docs/content/components/tabs

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -90,6 +116,14 @@ pnpm build
 
 Add a `.env` file in `/apps/agent/.env` and populate the values from the services above.:
 
+### Windows
```
Comment (chronark) [inline .github/CONTRIBUTING.md:119]: tabs please
https://mintlify.com/docs/content/components/tabs

**Review by chronark — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (KartikSoneji) [reply .github/CONTRIBUTING.md:85]: Hmm that completely breaks on Github.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (KartikSoneji) [reply .github/CONTRIBUTING.md:53]: Oh it is? I thought `ssl` was a client security option.
`?` can cause issues on Windows PowerShell, so I thought best to remove it.
If it's needed I'll have to do a little testing to find a command that works in both cmd and PowerShell.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (KartikSoneji) [reply .github/CONTRIBUTING.md:85]: If we're doing tabs, can they switch thought the page instead of having to click each one?
Or even better, switch automatically based on the user agent?
If neither's possible, maybe it's better to duplicate the entire process and have one tab at the beginning.

**Comment by chronark** [comment]
@KartikSoneji ah sorry, you are correct it breaks github
this should be updated [here](https://github.com/unkeyed/unkey/blob/main/apps/docs/contributing/getting-started.mdx)
I have already removed the CONTRIBUTING.md as it was a duplicate

**Comment by KartikSoneji** [comment]
OK, any chance of the auto-switching or global switching?

**Comment by chronark** [comment]
I don't think they support that.

**Comment by KartikSoneji** [comment]
Hmm it's a webpage at the end of the day, right?
Can I add an inline script to do that?

**Comment by chronark** [comment]
I don't think it's possible and please don't overcomplicate things
it's just 1 click to toggle the OS

**Review by chronark — APPROVED** [review]
lgtm
